examples/vision/learnable_resizer.py

```diff
@@ -202,7 +201,7 @@ def get_model():
     backbone = tf.keras.applications.DenseNet(weights=None, include_top=True, classes=2)
```

Comment:
I think it won't work, `tf.keras.application` has no function `DenseNet`

[densenet](https://www.tensorflow.org/api_docs/python/tf/keras/applications/densenet),

also when you use the given densenet variants you would have to use `input_shape` as

 ( `target_size[0], target_size[1],3)`

Reply:
Yes. Thanks for pointing these out. Corrected.